exonum/src/crypto.rs

```diff
@@ -424,7 +526,19 @@ implement_public_sodium_wrapper! {
 }
 
 implement_private_sodium_wrapper! {
-/// Ed25519 seed that can be used for deterministic keypair generation.
+/// Ed25519 seed that can be used for deterministic keypair generation. The
+/// seed is a succession of bytes which can be used to generate a certain
+/// keypair. If the same seed is indicated in the generator multiple times,
```

Comment:
> +/// Ed25519 seed that can be used for deterministic keypair generation. The seed is a succession of bytes which can be used to generate a certain keypair.

Let's unite these sentences into one: "Ed25519 seed representing a succession of bytes that can be used for deterministic keypair generation.